### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

**Review by cxw42 — APPROVED** [review]

**Comment by cxw42** [comment]
Thanks for the contribution!  Looks good to me; I just want to hear back from @xuhdev in https://github.com/editorconfig/editorconfig/issues/360#issuecomment-1368505205 before merging since I wasn't part of the `spec.editorconfig.org` discussion.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # EditorConfig Specification
 
 This repository hosts the specification of EditorConfig. A built version is
-available at https://editorconfig-specification.readthedocs.io/ .
+available at <https://spec.editorconfig.org/>.
```
Comment (xuhdev) [inline README.md:4]: I don't think this will display properly with `<>`. Could you remove them?

**Review by xuhdev — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (xuhdev) [reply README.md:4]: nvm, thought this was the website

**Review by xuhdev — APPROVED** [review]

**Comment by xuhdev** [comment]
Thank you for your contribution!

**Pull request merged**